package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@fiverr/uploader",
```

Comment:
It's okay to namespace open packages. It removed name collision dangers and encourages brand awareness.